webapp/src/Utils/Utils.php

```diff
@@ -1004,18 +1004,36 @@ public static function wrapUnquoted(string $text, int $width = 75, string $quote
     }
 
     /**
-     * Generate a random password of length 6 with lowercase alphanumeric
-     * characters, except o, 0, l and 1 since these can be confusing.
+     * Generate a random password.
+     *
+     * When $moreEntropy is `true`, generate a password of length 16 with
+     * alphanumeric characters and _ and -. When `false`, generate a password of
+     * length 6 with lowercase alphanumeric, except o, 0, l and 1. `false`
+     * should be used when generating password that will be printed and handed
+     * out. In other cases, use `true`.
+     *
+     * @param bool $moreEntropy
+     *
+     * @return string
      */
-    public static function generatePassword() : string
+    public static function generatePassword($moreEntropy = true) : string
```

Comment:
Why no type hint?

Reply:
Since I work too much 🤣 (we still have some projects that don’t allow type hints). I’ll fix it